tests/integration/targets/dcnm_vrf/tests/dcnm/query.yaml

```diff
@@ -88,11 +88,9 @@
     - 'result.response[0].attach[0].switchDetailsList[0].islanAttached == true'
     - 'result.response[0].attach[0].switchDetailsList[0].lanAttachedState == "DEPLOYED"'
     - 'result.response[0].attach[0].switchDetailsList[0].vlan == 500'
-    - 'result.response[0].attach[0].switchDetailsList[0].extensionValues == ""'
```

Comment:
Why are we removing this check?

Reply:
This check is being removed as in case of multisite fabrics, the extensionValues field will not be empty. It will have some values. So in order to make same playbook work for both normal and multisite fabric, this check has been removed.